src/quo2/foundations/colors.cljs

```diff
@@ -215,7 +215,7 @@
                60 "#CC6438"}
    :army      {50 "#216266"
                60 "#1A4E52"}
-   :pink      {50 "#F66F8F"
+   :flamingo  {50 "#F66F8F"
```

Comment:
This may cause issues to the existing accounts which use `pink`. We might need to inform the QA about this.

Reply:
Hey @status-im/mobile-qa, can somebody check this please? An account which uses color `pink` already would be affected by this PR's build?

Reply:
looks like this list in status-go is wrong. Is there a primary color? 🤔